book/generate-index.R

```diff
@@ -39,7 +39,7 @@ cat(
 section_header("Tables")
 create_subsection("./tables/ada", "ADA")
```

Comment:
Also change the other 3 to uppercase
```suggestion
create_subsection("./tables/ADA", "ADA")
```